OrbitGl/TracepointTrack.cpp

```diff
@@ -31,3 +65,25 @@ void TracepointTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, Pic
     }
   }
 }
+
+void TracepointTrack::SetPos(float x, float y) {
+  y = y - GetHeight();
+  pos_ = Vec2(x, y);
+
+  thread_name_.SetPos(pos_);
+  thread_name_.SetSize(Vec2(size_[0] * 0.3f, size_[1]));
+}
+
+float TracepointTrack::GetHeight() const {
+  TimeGraphLayout& layout = time_graph_->GetLayout();
+
+  return has_tracepoints_ == true
+             ? layout.GetEventTrackHeight() + layout.GetSpaceBetweenTracksAndThread()
+             : 0;
+}
+
+void TracepointTrack::HasTracepoints() {
+  if (!has_tracepoints_) {
```

Comment:
Caching this in a member variable seems a bit dangerous. Is this really needed? I might miss something here...

Consider providing a function
```
bool TracepointTrack::HasTracepoints() {
  return !GOrbitApp->GetCaptureData().GetTracepointsOfThread(thread_id_).empty();
}
```
and just call this every time (it's only twice - in GetHeight and Draw).

Reply:
Sure. Thank you!